### DISCUSSION TITLE
Icon editor needs refactoring

### DISCUSSION THREAD

**Discussion by svelderrainruiz** [ni/labview-icon-editor#92] [Ideas]
Leaving a placeholder here to start the discussion around re architecting the icon editor

**Comment by pettaa123** [comment]
Sorry for just crashing into this. I don't know if something similar was already crafted into new icon editor. I would like to see customizable behaviour.
E.g. by injecting ones own class from LabVIEW Data via redirceting an .ini path.

**Comment by crossrulz** [comment]
This is too broad of a discussion idea. If there are specific things to refactor, which there are plenty of, those specific cases should be their own discussion.
